controller/instance_handler.go

```diff
@@ -166,6 +171,18 @@ func (h *InstanceHandler) getPod(podName string) (*v1.Pod, error) {
 	return h.pLister.Pods(h.namespace).Get(podName)
 }
 
+func (h *InstanceHandler) getPodLogs(podName string, taillines int) ([]byte, error) {
+	tails := int64(taillines)
+	req := h.kubeClient.CoreV1().Pods(h.namespace).GetLogs(podName, &v1.PodLogOptions{
+		Timestamps: true,
+		TailLines:  &tails,
+	})
+	if req.URL().Path != "" {
+		return req.DoRaw()
+	}
+	return nil, errors.New("GetLogs request's path error")
```

Comment:
Use `fmt.Errorf` instead, which we have already imported. `errors` are mostly used with `Wrap` family functions.